conftest.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+def pytest_addoption(parser):
+    parser.addoption('--update-testcases', action='store_true',
+                     dest='UPDATE_TESTCASES')
+
+# mypy.test.helpers defines several top-level utility functions that
+# pytest will pick up as tests. This removes them.
```

Comment:
Hmm. How about just renaming those functions? Is it just `testfile_pyversion` and `testcase_pyversion`? This logic is complex enough that I worry about it being fragile.